webinarium/symfony-lazysec/2.0/config/packages/lazysec.yaml

```diff
@@ -0,0 +1,8 @@
+services:
+    _defaults:
+        autowire:      true
```

Comment:
Do you really need _defaults and autoworker when you only congure one service?

(I have not looked at the bundle, maybe you do)

Reply:
Thank you for your feedback. You are absolutely right - no need for the defaults, so I have simplified the config file and made another commit.